flink-table/flink-table-planner-blink/src/test/java/org/apache/flink/table/planner/operations/SqlToOperationConverterTest.java

```diff
@@ -575,11 +584,15 @@ public void testAlterTable() throws Exception {
 			assertEquals(expectedNewIdentifier, alterTableRenameOperation.getNewTableIdentifier());
 		}
 		// test alter table properties
-		Operation operation = parse("alter table cat1.db1.tb1 set ('k1' = 'v1', 'k2' = 'v2')", SqlDialect.DEFAULT);
+		Operation operation = parse("alter table cat1.db1.tb1 set ('k1' = 'v1', 'K2' = 'V2')", SqlDialect.DEFAULT);
 		assert operation instanceof AlterTablePropertiesOperation;
 		final AlterTablePropertiesOperation alterTablePropertiesOperation = (AlterTablePropertiesOperation) operation;
 		assertEquals(expectedIdentifier, alterTablePropertiesOperation.getTableIdentifier());
 		assertEquals(2, alterTablePropertiesOperation.getCatalogTable().getProperties().size());
+		Map<String, String> properties = new HashMap<>();
+		properties.put("k1", "v1");
```

Comment:
You can add new key-value pairs instead of modifying the existing ones.

Reply:
let me have a think...